test/suites/clustering.sh

```diff
@@ -277,9 +277,13 @@ test_clustering_containers() {
   LXD_DIR="${LXD_TWO_DIR}" lxc info bar | grep -q "Location: node3"
 
   # Move the container on node3 to node1, using a client connected to
-  # node2.
+  # node2 and a different container name than the original one.
   LXD_DIR="${LXD_TWO_DIR}" lxc move bar egg --target node1
   LXD_DIR="${LXD_ONE_DIR}" lxc info egg | grep -q "Location: node1"
+
+  # Move back to node3 the container on node1, keeping the same name.
+  LXD_DIR="${LXD_TWO_DIR}" lxc move egg --target node3
+  LXD_DIR="${LXD_ONE_DIR}" lxc info egg | grep -q "Location: node3"
```

Comment:
Might be worth checking volatile.apply_template in this test too

Reply:
Fixed.